lib/components/chip.ts

```diff
@@ -140,6 +148,9 @@ export const chipProps = commonComponentProps.extend({
   schPinSpacing: distance.optional(),
   schWidth: distance.optional(),
   schHeight: distance.optional(),
+  schRefAnchorPosition: point.optional(),
+  schRefAnchorAlignment: ninePointAnchor.optional(),
```

Comment:
do we need `schRefInsideBox={true}`? Because they're normally outside the box right

Reply:
hmm i think there's an issue, it should really be schRefAnchorOffset and be relative to component center- otherwise it won't work with autolayout etc.

I see your point- i was thinking this was more like `schRefAlignment`